lib/muffin_man/catalog_items/base_api.rb

```diff
@@ -33,7 +33,12 @@ def search_catalog_items(keywords, marketplace_ids, params = {}, api_version=API
         @query_params = {
           "marketplaceIds" => @marketplace_ids.join(",")
         }
-        @query_params["keywords"] = @keywords.join(",") if @keywords.any?
+        if params.key?("identifiers")
+          @query_params["identifiers"] = @keywords.join(",") if @keywords.any?
+          @query_params["identifiersType"] = @query_params["identifiersType"] || "ASIN"
+        else
+          @query_params["keywords"] = @keywords.join(",") if @keywords.any?
```

Comment:
Why make this change? Can't you just do 
```
params = {
  "keywords" => [],
  "identifiers" => "ASIN1, ASIN2",
  "identifiersType" => "ASIN"
}
```
And get the same result?

Reply:
@gavinritchie Motivation behind the change is for consistency in the calls to the endpoint. You are correct, the param body you gave as an example would work and get the desired result. But the Sp-Api endpoint we are editing here takes either a list of keywords, or a list of identifiers, and the Gem should be able to receive either kind of list in the same way (ie as an array being passed, and not forced to .join(',') on the identifiers list before sending it).

Reply:
Why not just do?
```
@query_params["identifiers"].join(",")
```
I don't love the implementation here because we're overwriting the "identifiers" keyword with what gets passed for "keywords", which could cause unintended confusion.